js/components/widgets/Image.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @function Image
+ * @typedef {Object} Data
+ * @property {string} type
+ * @property {url} type
+ * @param {Data} data
+ * @param {string} parrentClassName
+ * @returns {string} HTML or empty
+ */
+
+export const Image = (data, parrentClassName) => {
+  if (!Object.keys(data).length === 2) return '';
+
+  const currentClassName = parrentClassName 
+    ? `${parrentClassName}__image` 
+    : `image`;
+
+    return `
+      <img
+        class="${currentClassName}"
+        src="${url}"
```

Comment:
Why url?
Where alt?

Reply:
Because the component data is received by url
(added alt)